virt/virt-additional-security-privileges-controller-and-launcher.adoc

```diff
@@ -21,6 +21,7 @@ include::modules/virt-additional-scc-for-kubevirt-controller.adoc[leveloffset=+1
 [id="additional-resources_{context}"]
 == Additional resources
 
```

Comment:
I realize that this is my fault, but there isn't supposed to be a blank line before the add'l resources list starts. If you don't mind, please remove it. :)